test/tests/ver-1dc/comparison_ver-1dc.py

```diff
@@ -0,0 +1,102 @@
+import matplotlib.pyplot as plt
+import numpy as np
+from matplotlib import gridspec
+import pandas as pd
+import os
+import git
+
+# Changes working directory to script directory (for consistent MooseDocs usage)
+os.chdir(os.path.dirname(__file__))
+
+# ============ Comparison of permeation as a function of time =================
+# ========================= Diffusion limited =================================
+num_summation_terms = 1000
+
+# Reference 1:
+# Verification and Validation of Tritium Transport code TMAP7
+# Glen Longhurst & James Ambrosek, Fusion Science & Technology 2017
+
+
+# Reference 2:
+# GR Longhurst, SL Harms, ES Marwil, and BG Miller. Verification and validation of
+# tmap4. Technical Report, EG and G Idaho, Inc., Idaho Falls, ID (United States), 1992.
```

Comment:
Since this case is from TMAP8, I don't understand why we need the reference for TMAP4.